k8s-resources.tf

```diff
@@ -55,5 +61,9 @@ resource "kubernetes_service_account" "alb_ingress_controller" {
       "eks.amazonaws.com/role-arn" = "arn:aws:iam::${var.account_id}:role/${aws_iam_role.alb-ingress-controller-iam-role.name}"
     }
   }
-  automount_service_account_token = true
+  secret {
+    name = "alb-ingress-controller"
```

Comment:
**question:** without having tested this: I wonder if this is not possible? 🤔 If this is possible we could remove the `depends_on` since we communicate the relation to Terraform.

```suggestion
    name = kubernetes_secret.alb_ingress_controller.metadata.name
```